client/src/pages/course/mentor/confirm.tsx

```diff
@@ -75,21 +78,65 @@ function Page(props: { session: Session; courseAlias?: string }) {
     }
   };
 
-  if (course == null || (mentorData == null && loading)) {
+  if (loading) {
     return null;
   }
 
+  if (course == null || (mentorData == null && loading)) {
+    return (
+      <PageLayout loading={loading} githubId={props.session.githubId}>
+        <Row justify="center" style={{ margin: '65px 0 25px 0' }}>
+          <Image src="/static/svg/err.svg" alt="Error 404" width={175} height={175} />
```

Comment:
We have some reusable component for 404, don't we?

Reply:
No, I propose to discuss this and do it not within the framework of this task